core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -54,7 +54,7 @@ import scala.util.control.ControlThrowable
  *       1. use new producer with following settings
  *            acks=all
  *            retries=max integer
- *            block.on.buffer.full=true
+ *            max.block.ms.config=max long
```

Comment:
Thanks for adding this. I would write the actual config name, not the variable name. Something like:

`max.block.ms=Long.MAX_VALUE`